app/mapper/UserMapper_test.py

```diff
@@ -24,14 +24,14 @@ def no_find(_):
 
 def test_find_not_found_in_id_map_found_in_DB(monkeypatch):
     # Test Data
-    expected = User(10, 'joe', 'shmoe')
+    expected = User(10, 'joe', 'shmoe', False)
```

Comment:
Yes, as suggested by Arek, when creating a user the capstone flag is set to flag when unspecified, but in this dummy data it can be either.